src/app/src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { IoIosHome } from 'react-icons/io';
+import {
+    Breadcrumb,
+    BreadcrumbItem,
+    BreadcrumbLink,
+    Box,
+    Button,
+} from '@chakra-ui/react';
+
+import { ROUTES } from '../utils/constants';
+
+const CustomBreadcrumb = ({ isCurrentPage, icon, title, to }) => (
+    <BreadcrumbItem spacing={1}>
```

Comment:
This is working really well, but I think some of the components need to be modified to remove an unexpected behavior. The `currentPage` version of this still behaves like a link (is hover/clickable and performs a navigation). After looking at the docs, it appears we should do the following:

- For `isCurrentPage` `BreadcrumbLink`s, don't apply the `as={Link}` prop (see https://chakra-ui.com/docs/components/breadcrumb#usage-with-routing-library)
- For `isCurrentPage` don't use a `Button` component, as it has style and interactivity to look like it should be clicked. I think you can just swap this right out with `Box` and maybe tweak the style
- For `isCurrentPage` for the Breadcrumb _Item_, we also need to apply the `isCurrentPage prop (based on the docs, it may not even be required on the `BreadcrumbLink` component).

I think it'll need to be refactored a little bit to get all of those in (ie, the conditionally returned component is probably the whole `BreadcrumbItem` but there may be ways to just selectively compose the subcomponents. If all of those changes are applied, I think the breadcrumb bar behaves as expected: any non-current `BreadcrumbItems` are clickable/interactive but the current one is not, it is basically just a label.

Reply:
I've updated the breadcrumbs to correctly handle the isCurrentPage prop. Thanks for the advice!